### PR TITLE
Added files needed to run the sample dashboard and updated field validations

### PR DESCRIPTION
**Pull request by NipunaMarcus**

### CODE REVIEW

File: apps/portal/jaggery.conf [hunk 1]
```diff
@@ -1,79 +1,80 @@
 {
```
Comment (sinthuja) [inline apps/portal/jaggery.conf]: Add license header.

File: apps/portal/js/designer.js [hunk 2]
```diff
@@ -1172,7 +1172,7 @@ $(function () {
         }
 
         if(sandbox.context.className == "form-control title"){
-            if(checkForPagesByTitle(title) && page.title != title){
+            if(checkForPagesByTitle(title) && page.title.toUpperCase() != title.toUpperCase()){
                 generateMessage("error","A page with entered title already exists. Please select a different title");
```
Comment (sinthuja) [inline apps/portal/js/designer.js]: Formatting issue. There should be a space for ','. And use locales without hard coding the messages.

Comment (NipunaMarcus) [inline apps/portal/js/designer.js]: Cannot use i18n in the java script.

File: apps/portal/jaggery.conf (hunk 1 above)
Comment (NipunaMarcus) [inline apps/portal/jaggery.conf]: we cannot add licence headers to a json data containing file as they only allow to contain json data.

**Pull request merged**